corehq/apps/accounting/migrations/0043_grandfather_case_privs.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.21 on 2019-07-23 16:43
+from __future__ import unicode_literals
+
+from __future__ import absolute_import
+from django.core.management import call_command
+from django.db import migrations
+
+from corehq.apps.hqadmin.management.commands.cchq_prbac_bootstrap import (
+    cchq_prbac_bootstrap,
+)
+from corehq.privileges import (
+    CASE_SHARING_GROUPS,
+    CHILD_CASES,
+)
+
+
+def _grandfather_case_privs(apps, schema_editor):
```

Comment:
If this doesn't need to be run when bootstrapping new environments or in tests, could you add the `@skip_on_fresh_install` decorator to it?